bindings/pyroot_experimental/PyROOT/src/TDirectoryPyz.cxx

```diff
@@ -35,11 +35,8 @@ PyObject *TDirectoryWriteObject(const CPPInstance *self, PyObject *args)
    PyObject *name = nullptr;
    PyObject *option = nullptr;
    Int_t bufsize = 0;
-   if (!PyArg_ParseTuple(args, const_cast<char *>("O!O!|O!i:TDirectory::WriteObject"),
-                         &CPPInstance_Type, &wrt,
-                         &CPyCppyy_PyUnicode_Type, &name,
-                         &CPyCppyy_PyUnicode_Type, &option,
-                         &bufsize))
+   if (!PyArg_ParseTuple(args, const_cast<char *>("O!O!|O!i:TDirectory::WriteObject"), &CPPInstance_Type, &wrt,
```

Comment:
This one I would prefer that you leave it as it was, without the clang-format changes (it is more readable).